doc/openapi/python/postprocess.py

```diff
@@ -20,49 +34,87 @@
     'PTZ': 'https://www.onvif.org/ver20/ptz/wsdl/ptz.wsdl'
 }
 
+# list of superfluous headers to remove from response objects
+HEADERS_TO_REMOVE = [
+    'Content-Length',
+    'Date',
+    'Transfer-Encoding'
+]
+
 
 class ProcessingError(RuntimeError):
     pass
 
 
-@dataclasses.dataclass
+def multiline_string(val: str):
+    """Takes a string value and wraps it so that ruamel.yaml will format it as a raw multi-line string"""
+    return LiteralScalarString(textwrap.dedent(val))
+
+
+def single_quote(s: str):
+    """Returns the input value wrapped in single quote marks"""
+    return f"'{s}'"
+
+
+@dataclass
 class YamlProcessor:
     input_file: str
     sidecar_file: str
     profile_file: str
     output_file: str
-    yml = None
-    sidecar = None
-    profile = None
-    resources = {}
+    matrix: MarkdownMatrix
+    postman_env_file: str
+    yml: any = None
+    sidecar: any = None
+    profile: any = None
+    postman_env: dict[str, str] = field(default_factory=dict)
```

Comment:
to get it to work, I had to add '

    postman_env: 'dict[str, str]' = field(default_factory=dict)
    resources: 'dict[str, any]' = field(default_factory=dict)
    wsdl_files: 'dict[str, any]' = field(default_factory=dict)